robusta_krr/strategies/simple.py

```diff
@@ -30,7 +29,11 @@ class SimpleStrategySettings(StrategySettings):
     points_required: int = pd.Field(
         100, ge=1, description="The number of data points required to make a recommendation for a resource."
     )
-    allow_hpa: bool = pd.Field(False, description="Whether to calculate recommendations even when there is an HPA scaler defined on that resource.")
+    allow_hpa: bool = pd.Field(
+        False,
+        param_decls="allow-hpa",
```

Comment:
It feels like it is a bit incorrect to have all parameters with `_` and one with `-`
Maybe a better thing to do is to have both? (for backwards-compatilibity) and then we can do some 
```.replace("_", "-")``` 
where we use those parameters, tho it will be a bit harder

Reply:
I did it before, but maybe it's better to make this change for major release?

Reply:
as it would be a BC

Reply:
ok, let me check if it's possible to have both options at the same time

Reply:
done